glacier_mapping/models/frame.py

```diff
@@ -49,7 +48,7 @@ def __init__(self, loss_fn=None, model_opts=None, optimizer_opts=None,
         optimizer_def = getattr(torch.optim, optimizer_opts.name)
         self.optimizer = optimizer_def(self.model.parameters(), **optimizer_opts.args)
         self.lrscheduler = ReduceLROnPlateau(self.optimizer, "min",
-                                             verbose=True, patience=500,
+                                             verbose=True, patience=10,
```

Comment:
I thought I just commented on this -- but isn't this is a small value to use for patience?